lib/vrl/stdlib/benches/benches.rs

```diff
@@ -133,10 +134,20 @@ criterion_group!(
               //unnest
               // TODO: value is dynamic so we cannot assert equality
               //uuidv4,
-              upcase
+              upcase,
+    bench_crypto
```

Comment:
I think this should be `encrypt`? We should add a benchmark for `decrypt` too.

Reply:
It might be interesting to have another bench for a larger event too to see how that compares. Maybe a 1 KiB one?

Reply:
This was leftover from some manual testing I added. I'm cleaning it up / adding a `decrypt`